### DISCUSSION TITLE
[Question] Peak Cache Of Wavesurfer Generated Thumbnail

### DISCUSSION THREAD

**Discussion by jakepenn** [katspaugh/wavesurfer.js#2681] [Q&A] [answered]
Hi! I've seen pieces of the docs for Peaks Cache where user generates the peaks themselves and then serves this to the wave surfer constructor -- I gave that a shot but my peaks were not looking nearly as good as the waveforms generated by wave surfer. 

Is there a way to obtain the peaks data from a wave surfer object which has generated the peaks? I.E. a way to get it so wave surfer only generates the thumbnails once -- else is able to obtain this from cache? 

Apologies if this is totally outside of the scope here

**Comment by entonbiba** [comment, marked as answer]
@jakemumu you can do that this way

``` javascript
wavesurfer.exportPCM();
```
